src/VoteDelegate.sol

```diff
@@ -15,18 +14,17 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-// VoteDelegate - delegate your vote
-pragma solidity 0.6.12;
+pragma solidity ^0.8.16;
 
-interface TokenLike {
+interface GemLike {
     function approve(address, uint256) external returns (bool);
-    function pull(address, uint256) external;
-    function push(address, uint256) external;
+    function transfer(address, uint256) external;
+    function transferFrom(address, address, uint256) external;
 }
 
 interface ChiefLike {
-    function GOV() external view returns (TokenLike);
-    function IOU() external view returns (TokenLike);
+    function GOV() external view returns (GemLike);
+    function IOU() external view returns (GemLike);
```

Comment:
Question about `withdrawPoll` - is it a function intended to be used by delegates or by whoever sets up the poll?
I'd think that if it is the first case, then it should be also part of `VoteDelegate` (and weird that it's not already), and if it's the second case, then it should be removed from the interface here.

Reply:
Removed.